lib/debug/thread_client.rb

```diff
@@ -15,6 +16,7 @@ def self.current
       end
     end
 
+    extend IRB::ExtendCommandBundle
```

Comment:
This module injects irb commands as methods. Pretty handy 👍 
https://github.com/ruby/irb/blob/master/lib/irb/extend-command.rb